packages/saltcorn-data/models/trigger.ts

```diff
@@ -301,6 +301,20 @@ class Trigger implements AbstractTrigger {
               }
             }
           } else {
+            if (payload?.type === "page" && trigger.configuration?._only_if) {
```

Comment:
why is this code special? there shouldnt be any special code here related to pages. What happens if you remove this and evaluatePageExpression ? what happens if you have a PageLoad event with an only-if expression?